scripts/reformat

```diff
@@ -6,6 +6,6 @@ set -euox pipefail
 echo 'Reformatting Rust code'
 cargo fmt
 echo 'Reformatting Python code'
-poetry run isort test_runner
-poetry run flake8 test_runner
-poetry run black test_runner
+poetry run isort test_runner scripts
+poetry run flake8 test_runner scripts
+poetry run black test_runner scripts
```

Comment:
Nice!
Didn't know about this script

Reply:
Linked to be today by @SomeoneToIgnore :)

Reply:
It's a bit of an odd name given that one of the tools fails on the non-formatted imports.
So I think we could rename it to `style_check` (?), add run_clippy.sh and a mypy check, but not sure yet, since those two are slower.